File: model/src/main/java/net/artux/pda/model/map/Strength.java
package net.artux.pda.model.map;

public enum Strength {
    WEAK("gang.strength.weak"),
    MIDDLE("gang.strength.middle"),
    STRONG("gang.strength.strong");

    private final String titleId;

    Strength(String titleId) {
        this.titleId = titleId;
    }

    public String getTitleId() {
        return titleId;
    }
}

File: model/src/main/java/net/artux/pda/model/map/Strength.kt
package net.artux.pda.model.map

enum class Strength(val titleId: String, val price: Int) {
    WEAK("gang.strength.weak", 0),
    STALKER("gang.strength.stalker", 5000),
    MIDDLE("gang.strength.middle", 10000),
    MASTER("gang.strength.master", 15000),
    STRONG("gang.strength.strong", 30000);

    fun getNext(): Strength {
        return when (this) {
            STALKER -> MIDDLE
            MIDDLE -> MASTER
            MASTER -> STRONG
            STRONG -> STRONG
            else -> STALKER
        }
    }
}